runtime/centrifuge/src/migrations.rs

```diff
@@ -169,4 +171,59 @@ mod asset_registry {
 			},
 		}
 	}
+
+	/// Register the LiquidityPools Wrapped Ethereum USDC
+	pub struct RegisterLpEthUSDC;
+
+	impl OnRuntimeUpgrade for RegisterLpEthUSDC {
+		fn on_runtime_upgrade() -> Weight {
+			use orml_traits::asset_registry::Mutate;
+
+			if OrmlAssetRegistry::metadata(&LP_ETH_USDC_CURRENCY_ID).is_some() {
+				log::info!("LpEthUSDC is already registered");
+				return RocksDbWeight::get().reads(1);
+			}
+
+			<OrmlAssetRegistry as Mutate>::register_asset(
+				Some(LP_ETH_USDC_CURRENCY_ID),
+				lp_eth_usdc_metadata(
+					LiquidityPoolsPalletIndex::get(),
+					// Ethereum mainnet chain id
+					1,
```

Comment:
Nit: If we know we might need this chain ID in future migrations as well, could we make this a primitive, i.e. `ETHEREUM_MAINNET_CHAIN_ID`?

Reply:
Done, thank you for the suggestion!